pkgs/os-specific/linux/busybox/default.nix

```diff
@@ -75,6 +75,26 @@ stdenv.mkDerivation rec {
       url = "https://git.alpinelinux.org/aports/plain/main/busybox/0002-nslookup-sanitize-all-printed-strings-with-printable.patch?id=ed92963eb55bbc8d938097b9ccb3e221a94653f4";
       sha256 = "sha256-vl1wPbsHtXY9naajjnTicQ7Uj3N+EQ8pRNnrdsiow+w=";
     })
+    (fetchurl {
```

Comment:
This one needs to be fetchpatch.  The response is dynamically generated by cgit (including a version number).

Reply:
Oups, yes indeed. Good catch thanks.

Reply:
Used `fetchpatch` for all the new patches since we are pulling it into the derivation anyway.